microcosm_flask/swagger/schemas.py

```diff
@@ -6,8 +6,10 @@
     Any,
     Callable,
     Iterable,
+    List as TypingList,
```

Comment:
Best practices here?  Conflicts with `marshmallow.fields.List` cc/ @marcglobality @GabeLambda

Reply:
`ListType`, maybe.

Reply:
in `microcosm_flask/tests/conventions/fixtures.py` they import like `from marshmallow import fields` and do `fields.List`. If you were doing the same here there would be no conflict. In any case I think it should be uniformised at least for the same repo, whichever pattern is decided

In general, I'm more inclined to have `{name_of_library}_{module}`. I had similar imports in the past as in

```
from sklearn.metrics import precision as sklearn_precision
from my_module import precision
```
so I'd prefer your original choice which is the analogous when using PascalCase